sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -595,6 +630,8 @@ func computeUpdatedConns(current map[networkConnIndicator]timestamp.MicroTS, pre
 		}
 	}
 
+	previousMutest.RLock()
```

Comment:
We do `prevTS, ok := previous[conn]` in line 627 - we need to lock this one earlier.

Reply:
Good catch! I fixed the other ones and completely missed this one.